core/src/main/java/bio/terra/pearl/core/service/kit/PepperDSMClientProvider.java

```diff
@@ -20,7 +20,7 @@ public PepperDSMClientProvider(LivePepperDSMClient livePepperDSMClient, StubPepp
     @Primary
     @Bean
     public PepperDSMClient getPepperDSMClient() {
-        return getStubPepperDSMClient();
+        return getLivePepperDSMClient();
```

Comment:
Update coming to make this configuration-based so it can default to stub for local dev and live for demo.